src/components/Button/Button.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Button from './Button.component';
+import renderer from 'react-test-renderer';
```

Comment:
I would suggest you use react-testing-library since it's the recommended approach